cvss/cvss3.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cvss
+
+import (
+	"fmt"
+
+	. "github.com/facebookincubator/nvdtools/cvss/internal"
+	. "github.com/facebookincubator/nvdtools/cvss/internalv3"
```

Comment:
That's not how `internal` supposed to work. you need sub-dirs in `internal/` directory. otherwise your `internalv3` is just a regular package

Reply:
I'm planning to do `s/internal/common/` and `s/internalv3/v3/`
And also create `v2` implementation.